README.md

```diff
@@ -182,13 +203,18 @@ Name | Official | Tested | Compatible | Patched
 
 ### Launchers
 
-Name  | Official | Tested     | Compatible | Patched
-------|----------|------------|------------|------------
-PCSX2 |          |            |            | 🇻 1.7.3256
-Steam | Dropped  | 1705108172 |            |
+Name  | Official    | Tested     | Compatible | Patched
+------|-------------|------------|------------|------------
+PCSX2 | 1.6.0       |            |            | 🇻 1.7.3256
+Steam               | Dropped    | 1705108172 |
+EA App              | Supported  |            |
+GOG Galaxy          | 1.2        |            |
+Epic Games Launcher | Supported  |            |
+
+Note: The Epic Games Launcher will drop support in June 2024.
 
 ### Games
 
-Name | Official | Tested | Compatible | Patched
------|----------|--------|------------|--------
-     |          |        |            |
+Name   | Official | Tested | Compatible | Patched
+-------|----------|--------|------------|--------
+ngrok  | Dropped  |        |            | 3 🇻
```

Comment:
Is this a game though?

Reply:
ngrok is a private tunneling service. i guess i put it in the games category as it is used quite often for game servers but it fits the developer tools / ides category far better, thus i have just moved it to that category. ngrok is targetted towards developers after all.